fdbserver/workloads/FuzzApiCorrectness.actor.cpp

```diff
@@ -262,12 +266,17 @@ struct FuzzApiCorrectnessWorkload : TestWorkload {
 
 	Future<Void> start(Database const& cx) override {
 		if (clientId == 0) {
-			return loadAndRun(this);
+			return loadAndRun(this, cx);
 		}
 		return Void();
 	}
 
-	Future<bool> check(Database const& cx) override { return success; }
+	Future<bool> check(Database const& cx) override {
+		if (useSystemKeys) { // there must be illegal access during data load
```

Comment:
Is it expected that we would get `illegalTenantAccess` if `writeSystemKeys == true`?